Logs.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace M3
+{
+    public class Logs
+    {
+        private List<string> _logs = new List<string> { "Event log ", "Server log", "System log ", "Change log", "Availability log ", "Resource log" };
+        public List<string> Currentlog = new List<string>();
```

Comment:
u can make code clearer by separationg public and private things with a empty string